modules/security/src/main/java/io/oasp/module/security/common/base/accesscontrol/AccessControlBasedAuthenticationProvider.java

```diff
@@ -23,31 +23,31 @@
 import org.springframework.web.context.request.RequestContextHolder;
 import org.springframework.web.context.request.ServletRequestAttributes;
 
+import io.oasp.module.security.common.api.accesscontrol.AbstractAccessControl;
 import io.oasp.module.security.common.api.accesscontrol.AccessControl;
 import io.oasp.module.security.common.api.accesscontrol.AccessControlProvider;
 import io.oasp.module.security.common.api.accesscontrol.PrincipalAccessControlProvider;
 
 /**
  * This is an implementation of {@link AbstractUserDetailsAuthenticationProvider} based on
  * {@link PrincipalAccessControlProvider} and {@link AccessControlProvider}.
+ *
  * @deprecated As of bug-fix release 2.1.2 the authentication mechanism changes. It is now based upon custom
  *             implementations of {@link UserDetailsService} in combination with {@link WebSecurityConfigurerAdapter}.
  *             For further information have a look at the sample application. <br/>
  *             <br/>
  *
  * @param <U> is the generic type of the {@link UserDetails} implementation used to bridge with spring-security.
- * @param
- *        <P>
- *        is the generic type of the {@link Principal} for internal user representation to bridge with
+ * @param <P> is the generic type of the {@link Principal} for internal user representation to bridge with
  *        {@link PrincipalAccessControlProvider}.
  *
  */
 @Deprecated
-public abstract class AbstractAccessControlBasedAuthenticationProvider<U extends UserDetails, P extends Principal>
+public abstract class AccessControlBasedAuthenticationProvider<U extends UserDetails, P extends Principal>
```

Comment:
Why is this no longer *abstract* ?

Also, it says deprecated on the previous version... is it still being used? if not, we should just delete it (And document in the migration page)